pycbc/workflow/core.py

```diff
@@ -182,6 +183,12 @@ def __init__(self, cp, name,
         # Determine the level at which output files should be kept
         self.update_current_retention_level(self.current_retention_level)
 
+        # Should I reuse this executable?
```

Comment:
Should we abandon having a tag in the exe name entirely if this is going to be the default? Would seem to be easier to just remove the tagging in the exe name so it's consistent.

Reply:
I'd like to keep this, because as things are, in pegasus dashboard, you can separate INSPIRAL jobs by ifo and full_data/injections (ie. the tags). I'd like to keep that.

Tags only appear in pegasus exe names for jobs that aren't told to reuse executables.

Tags are still important in general in the Executable class for file naming.